src/Response/SapiStreamEmitter.php

```diff
@@ -78,21 +79,14 @@ private function emitBodyRange(array $range, ResponseInterface $response, $maxBu
     {
         list($unit, $first, $last, $length) = $range;
 
-        ++$last; //zero-based position
-        $body = $response->getBody();
-
-        if (!$body->isSeekable()) {
-            $contents = $body->getContents();
-            echo substr($contents, $first, $last - $first);
-            return;
-        }
-
-        $body->seek($first);
-        $pos = $first;
+        $body = new RelativeStream($response->getBody(), $first);
+        $body->rewind();
```

Comment:
This directly conflicts with #200 (represented in lines 84 - 88 above), which provided a fix that ensures that the emitter can be used with non-seekable streams. Please reinstate that block.